package.json

```diff
@@ -23,17 +23,17 @@
   },
   "devDependencies": {
     "@types/benchmark": "^1.0.31",
-    "@types/jest": "^24.0.0",
-    "@types/node": "^11.9.0",
+    "@types/jest": "^24.0.9",
+    "@types/node": "^11.10.4",
     "@types/pngjs": "^3.3.0",
     "benchmark": "^2.1.4",
-    "jest": "^23.6.0",
+    "jest": "^24.1.0",
     "microtime": "^3.0.0",
     "pngjs": "^3.3.3",
     "ts-jest": "^24.0.0",
     "ts-node": "^8.0.2",
-    "tslint": "^5.11.0",
-    "tslint-config-airbnb": "^5.1.2",
-    "typescript": "^3.0.3"
+    "tslint": "^5.13.1",
+    "tslint-config-airbnb": "^5.11.1",
+    "typescript": "^3.3.3333"
```

Comment:
this version looks weird :))

Reply:
Yeah I was confused at first, think they're just having some fun with version numbers! 😄